src/SharpCompress/Utility.cs

```diff
@@ -229,15 +229,18 @@ public static DateTime DosDateToDateTime(Int32 iTime)
             return DosDateToDateTime((UInt32)iTime);
         }
 
-        public static long TransferTo(this Stream source, Stream destination)
+        public static long TransferTo(this Stream source, Stream destination, Action<long, int> action = null)
```

Comment:
As I stated, my thought process was to not introduce a new dependency into utility, but if you are fine with this I will change to that behavior.